.storybook/preview.js

```diff
@@ -4,6 +4,7 @@ import { GlobalStyles } from 'twin.macro';
 
 import { useThemeVariables } from '$/contexts/ThemeProvider/useThemeVariables';
 import { appGlobalStyles } from '$/styles/global-styles';
+import 'tailwindcss/tailwind.css';
```

Comment:
This line is needed. Otherwise we lose styles:

https://user-images.githubusercontent.com/43892874/149960397-14969307-e4de-4714-8d9a-8d7b348e8349.mov